.gitmodules

```diff
@@ -2,3 +2,6 @@
     path = tests/fixtures
     url = https://github.com/ethereum/tests
     branch = develop
+[submodule "tests/execution-spec-generated-tests"]
+	path = tests/execution-spec-generated-tests
+	url = https://github.com/petertdavies/execution-spec-generated-tests.git
```

Comment:
Should we move this directly into this repository?

Reply:
See #699